src/screens/AlarmsScreen.tsx

```diff
@@ -1,72 +1,53 @@
 import * as React from 'react';
-import { View, StyleSheet } from 'react-native';
-import { ScrollView } from 'react-native-gesture-handler';
-import OptionButton from '../components/OptionButton';
+import { View, StyleSheet, ScrollView } from 'react-native';
 import DetailedAlarmMetricDisplay from '../components/DetailedAlarmMetricDisplay';
-import Layout from '../constants/Layout';
-import { useReading } from '../logic/useReading';
+import { listToMatrix } from '../utils/helpers';
+import { alarmsMetrics } from '../../sample-data/data';
+import { AlarmsMetrics } from 'src/lib/interfaces';
 
 export default function AlarmsScreen() {
-  const reading = useReading();
+  const metrics = listToMatrix(alarmsMetrics, 4);
```

Comment:
Could you explain what you are doing here?

Reply:
The data we are being received in terms of metrics is being broken into two dimensional array so that rows with metrics can be returned.